RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -420,6 +427,25 @@ public struct Home: Sendable, FeatureReducer {
 			}
 		}
 	}
+
+	public func fetchAccountPortfolios(_ state: State) -> Effect<Action> {
+		let accountAddresses = state.accounts.map(\.address)
+		return .run { _ in
+			_ = try await accountPortfoliosClient.fetchAccountPortfolios(accountAddresses, true)
+		} catch: { error, _ in
+			errorQueue.schedule(error)
+		}
+	}
+
+	public func scheduleFetchAccountPortfoliosTimer(_ state: State) -> Effect<Action> {
```

Comment:
hmm, need to test it, but this seems that it will start the automatic refresh for the accounts that are in the state at the moment of the task being fired.
What about the case when new accounts are added/hidden? Gateway switch?

Do check the `accountsLoadedResult` internal action.

Reply:
Good point. Starting the timer on the `accountsLoadedResult` action (and also on the `task` action to ensure it resumes when coming back) will fix the problem.